packages/grafana-ui/src/components/DataLinks/DataLinkEditor.tsx

```diff
@@ -55,6 +44,7 @@ export const DataLinkEditor = memo(
       <div className={styles.listItem}>
         <Field label={t('grafana-ui.data-link-editor.title-label', 'Title')}>
           <Input
+            id="link-title"
```

Comment:
just to confirm my understanding here, the reason we care about this from an a11y perspective is because the [getChildId method](https://github.com/grafana/grafana/blob/704743200be0560d1ca8e5c406bc3139564736a4/packages/grafana-ui/src/utils/reactUtils.ts#L5-L20) needs the `id` or `inputId` to be set on this child element under a `Field` to hook up the `htmlFor` correctly? i.e., the automatic id creation that `Switch` is doing internally won't hook up correctly?